lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
+  "connectionString": "${file:/mnt/secrets/pcq/app-insights-connection-string}",
```

Comment:
is this (`/mnt/secrets/pcq/`) where secrets are being mounted?

Reply:
Yes